R/corrpart.b.R

```diff
@@ -26,26 +25,19 @@ corrPartClass <- R6::R6Class(
             controls <- self$options$controls
             type <- self$options$type
 
-            if (hyp == 'pos')
-                hyp <- 'greater'
-            else if (hyp == 'neg')
-                hyp <- 'less'
-            else
-                hyp <- 'equal'
-
             results <- list()
             if (nVars > 1) {
                 for (i in 1:nVars) {
                     rowVar <- vars[[i]]
                     for (j in 1:nVars) {
-                        if (i == j || (type != 'semi' && j > i))
-                            next
+                        if (j >= i) { next }
                         colVar <- vars[[j]]
-                        results[[rowVar]][[colVar]] <- private$.test(rowVar, colVar, controls, type)
+                        resLst <- private$.test(rowVar, colVar, controls, type, hyp)
+                        results[[rowVar]][[colVar]] <- resLst[, 1]
+                        if (type == 'semi') { results[[colVar]][[rowVar]] <- resLst[, 2] }
```

Comment:
Same as before regarding the inline statements. Maybe even make it an ifelse statement to be more verbose (now you're assigning to results twice if type == 'semi'). 

So:
```
results[[rowVar]][[colVar]] <- ifelse(type == 'part', resLst[, 1], resLst[, 2])
```

Reply:
Resolved.
I am uncertain if your suggestion with ifelse works. results[[colVar]][[rowVar]] has to be assigned either way, it is only for results[[rowVar]][[colVar]] and semi-partial where the second row of resLst has to be assigned. But maybe I got you wrong.